### ISSUE TITLE
Failed to activate the keystroke package

### ISSUE THREAD

**Issue by Riddlerrr** [t9md/atom-keystroke#8]
**Atom**: 1.25.0 x64
**Electron**: 1.7.11
**OS**: Mac OS X 10.13.3
**Thrown From**: [keystroke](https://github.com/t9md/atom-keystroke) package 0.2.1

### Stack Trace

Failed to activate the keystroke package

```
At Cannot find module 'undefined/node_modules/atom-keymap/lib/helpers'

Error: Cannot find module 'undefined/node_modules/atom-keymap/lib/helpers'
    at Module._resolveFilename (module.js:470:15)
    at Module._resolveFilename (/Applications/Atom.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app/src/module-cache.js:354:58)
    at Module.require (/app.asar/static/index.js:40:43)
    at require (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:66:33)
    at /packages/keystroke/lib/utils.js:1:186)
    at /packages/keystroke/lib/utils.js:42:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/src/compile-cache.js:240:29)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:66:33)
    at /packages/keystroke/lib/main.js:2:29)
    at /packages/keystroke/lib/main.js:70:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/src/compile-cache.js:240:29)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (internal/module.js:20:19)
    at customRequire (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:96:26)
    at Package.requireMainModule (/Applications/Atom.app/Contents/Resources/app/src/package.js:782:33)
    at Package.activateNow (/Applications/Atom.app/Contents/Resources/app/src/package.js:220:40)
    at measure (/Applications/Atom.app/Contents/Resources/app/src/package.js:206:33)
    at Package.measure (/Applications/Atom.app/Contents/Resources/app/src/package.js:88:25)
    at activationPromise.Promise (/Applications/Atom.app/Contents/Resources/app/src/package.js:200:20)
    at Promise (<anonymous>)
```

### Commands

```
```

### Non-Core Packages

```
busy-signal 1.4.3 
ex-mode 0.18.0 
file-icons 2.1.17 
gruvbox 1.0.7 
highlight-selected 0.13.1 
ink 0.8.3 
intentions 1.1.5 
keystroke 0.2.1 
language-pgsql 0.2.3 
language-slim 1.0.0 
linter 2.2.0 
linter-coffeelint 1.3.1 
linter-rubocop 2.2.3 
linter-slim 1.0.0 
linter-ui-default 1.7.1 
lisp-paredit 0.6.0 
parinfer 1.22.0 
platformio-ide-terminal 2.8.0 
proto-repl 1.4.22 
proto-repl-charts 0.4.1 
rails-latest-migration 1.1.6 
ruby-test 1.0.2 
set-syntax 0.4.0 
sync-settings 0.8.5 
tab-numbers 0.6.1 
tool-bar 1.1.6 
trailing-spaces 0.4.0 
vim-mode-plus 1.30.1 
vim-mode-plus-keymaps-for-surround 0.2.1 
```

**Comment by calebmeyer** [comment]
Also seeing this on Windows 10.

**Comment by t9md** [comment]
Will fix from now. Sorry for unresponsiveness.

**Comment by t9md** [comment]
released v0.3.0.
